apis/logging/v1/index_management_types.go

```diff
@@ -49,6 +49,18 @@ type IndexManagementDeletePhaseSpec struct {
 	// The minimum age of an index before it should be deleted (e.g. 10d)
 	//
 	MinAge TimeUnit `json:"minAge"`
+	// +nullable
+	Namespace []DeleteNamespaceSpec `json:"deleteNamespace,omitempty"` //#BEE: Why is this an array?
```

Comment:
I believe the `array` for `Namespace` means that you can define a delete-by-query spec per namespace, because you can define a different `MinAge` per `Namespace`. @sasagarw Do you agree? If yes, we should improve the API naming because if it is confusing to us, it will certainly be confusing for API consumers.

```go
type IndexManagementDeletePhaseSpec struct {
	// The minimum age of an index before it should be deleted (e.g. 10d)
	//
	MinAge TimeUnit `json:"minAge"`

	// The per namesapce specification to delete documents older than a given minimum age.
	Namespaces []IndeManagementDeleteNamespaceSpec `json:"namespaces,omitempty"`
}
```

Reply:
Yes @periklis , you are right.
Yeah we can rename it to `Namespaces` to avoid confusion.